core/local/analysis.js

```diff
@@ -273,8 +273,10 @@ function squashMoves (changes /*: LocalChange[] */) {
         log.debug({oldpath: b.old.path, path: b.path}, 'descendant move')
         a.wip = a.wip || b.wip
         if (b.path.substr(a.path.length) === b.old.path.substr(a.old.path.length)) {
+          log.debug({oldpath: b.old.path, path: b.path}, 'ignoring move')
```

Comment:
Maybe explain why in log message?

Reply:
I actually didn't spend time to try and understand why we're ignoring it.
I'll do this work.